CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Release notes for 2.4.1 can be found on the [release notes page](https://grafana
 * [4683](https://github.com/grafana/loki/pull/4683) **owen-d**: Adds replication_factor doc to common config
 * [4681](https://github.com/grafana/loki/pull/4681) **slim-bean**: Loki: check new Read target when initializing boltdb-shipper store
 * [4736](https://github.com/grafana/loki/pull/4736) **sandeepsukhani**: allow applying retention at different interval than compaction
+* [4744](https://github.com/grafana/loki/pull/4744) **cyriltovena**: Promtail: Adds GELF UDP support.
```

Comment:
These got accidentally added in the  2.4.1 release notes, can you move them up above under `main`?